src/ucp/api/ucp.h

```diff
@@ -276,6 +279,65 @@ enum {
 
 /**
  * @ingroup UCP_COMM
+ * @brief Modificators for @ref ucp_stream_send_nb function.
+ *
+ * This enumeration defines changes of @ref ucp_stream_send_nb function behavior.
+ */
+enum {
+    UCP_STREAM_SEND_FLAG_MSG_MORE = UCS_BIT(0) /**< Hint for coaleasing inside of
```

Comment:
can we submit the send flags in another PR? this is not essential for the first phase